src/config.cc

```diff
@@ -0,0 +1,75 @@
+
+// The MIT License (MIT)
```

Comment:
any chance we could move this to the pr description instead of the code? I'd prefer not to have the bloat in the code especially for something so trivial